chapters/equations.tex

```diff
@@ -753,11 +753,17 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 In case of iterative solver failure, it is recommended to specially report those variables for which the solver needs an initial guess, but which only have the default value of the \lstinline!start!-attribute as defined in \cref{predefined-types-and-classes}, since the lack of appropriate initial guesses is a likely cause of the solver failure.
 \end{nonnormative}
 
-If a parameter has a modifier for the \lstinline!start!-attribute, does not have \lstinline!fixed = false!, and neither has a binding equation nor is part of a record having a binding equation, the modifier for the \lstinline!start!-attribute can be used to add a parameter binding equation assigning the parameter to that \lstinline!start! value.
-In this case a diagnostic message is recommended in a simulation model.
+A parameter not having \lstinline!fixed = false! and neither having a binding equation nor being part of a record having a binding equation, is handled in different ways depending on the presence of a modifier for the \lstinline!start!-attribute.
+If there is no modifier for the \lstinline!start!-attribute, it is an error unless a parameter value is explicitly provided before solving the initialization problem.
+In this case, a tool may give a diagnostic message when translating a simulation model to inform the user that additional parameter values must be provided later.
+If there is a modifier for the \lstinline!start!-attribute, the modifier can be used to add a parameter binding equation assigning the parameter to that \lstinline!start! value.
+In this case, a diagnostic message is recommended in a simulation model.
 
 \begin{nonnormative}
-This is used in libraries to give non-zero defaults so that users can quickly combine models and simulate without setting parameters; but still easily find the parameters that need to be set.
+The case of no \lstinline!start!-attribute is useful when no good default value for the parameter is known, or when the modeler wants to force the user of the model to make an active choice of parameter value.
+Note that the requirement must not be fulfilled by defaulting to something like the default value for the parameter's type, as this would bypass the intention that all parameters should be determined either by the model itself or by the user of the translated model.
+
+The case of providing a \lstinline!start!-attribute is used in libraries to give non-zero defaults so that users can quickly combine models and simulate without setting parameters; but still easily find the parameters that need to be set.
```

Comment:
```suggestion
The case of providing a \lstinline!start!-attribute is used in libraries to give defaults so that users can quickly combine models and simulate without setting parameters; but still easily find the parameters that need to be set.
```
Using zero as a fall-back for parameters is something we want to explicitly forbid in this proposal, or as a minimum see as less correct than a provided start-value.

This implies that the defaults in the library can also be zero - when it makes sense. 
Obviously a zero value can in many cases be used as an actual parameter-value, e.g., alpha in Modelica.Electrical.Analog.Basic.Resistor - but using a start-value of 0 is used in e.g., Modelica.Electrical.QuasiStatic.SinglePhase.Basic.Admittance.

Reply:
That's right.  Applying suggestion.